index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" >
+</head>
+<body>
+    <div id="container">
+        <h1 id="title">RashidRachel Hotel Survey Form</h1>
+        <form id="survey-form" action="Submit">
+            <p id="description">Thank You For Using RashidRachel Hotel! Please Let Us Know How We Can Improve Our
+                Services Down Below: 
+                </p>
+
+            <div class="row">
+                <div class="left">
+                    <label id="name-label" for="name">Name:</label>
+                </div>
+                <div class="Right">
+                    <input autofocus type="text" id="name" placeholder="Enter Your Name" required>
+                </div>
+
+            </div>
```

Comment:
- please remove ❌  blank lines in your code 👍

Reply:
- [ ] Good job by implementing some of the changes requested by the previous reviewer But please always tell the reason, **WHY** you suggest something or request a change. An example will be:  "Please remove x blank lines in your code .... because it's bad practice..."  I would like to recommend you check on google to have a solid reason.  Please fix it :smile: